packages/lib/server/getLuckyUser.ts

```diff
@@ -23,10 +39,11 @@ async function leastRecentlyBookedUser<T extends Pick<User, "id" | "email">>({
           createdAt: true,
         },
         where: {
-          eventTypeId,
+          eventTypeId: eventType.id,
+          status: BookingStatus.ACCEPTED,
```

Comment:
I think it makes more sense to only count ACCEPTED bookings also for `leastRecentlyBookedUser`. We maybe could also allow PENDING here, but canceled or rejected bookings shouldn't count

Reply:
@emrysal what do you think of that change?

Reply:
Sounds sensible! Yeah happy with that.